### PR TITLE
new package: mathematica

### PR DESCRIPTION
**Pull request by dkwo**
[ci skip]
This requires a proprietary script to run on x86_64, to be placed in ${FILESDIR}.
I'm not sure it is appropriate to have in the repos.
I welcome comments on possible stupid things I'm doing, in case others have tried installing Wolfram Mathematica through XBPS.

### CODE REVIEW

File: srcpkgs/mathematica/template [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Template file for 'mathematica'
+pkgname=mathematica
+version=13.0.0
+revision=1
+archs="x86_64"
+# depends="libfont fontconfig alsa-lib libXext libglvnd xdg-user-dirs"
+makedepends="tar xz"
```
Comment (Chocimier) [inline srcpkgs/mathematica/template:7]: These are hostmakedepends.

File: srcpkgs/mathematica/template [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		-targetdir=${DESTDIR}/opt/Mathematica \
+		-auto
+}
+
+do_install() {
+	# fix permissions
+	chmod -R go-w ${DESTDIR}/*
+	vlicense ${DESTDIR}/opt/Mathematica/LICENSE.txt
+	# remove docs
+	rm -rf ${DESTDIR}/opt/Mathematica/Documentation
```
Comment (Chocimier) [inline srcpkgs/mathematica/template:36]: Why to remove docs?

**Review by Chocimier — COMMENTED** [review]
Of course we cannot put proprietary installer into tree.

If it is meant for people to provide their own installer, it is at edge of being acceptable.
But then it should error out with helpful message when file is not there.

File: srcpkgs/mathematica/template (hunk 2 above)
Comment (dkwo) [reply srcpkgs/mathematica/template]: Just to save space. I made it an option.

**Comment by dkwo** [comment]
> If it is meant for people to provide their own installer, it is at edge of being acceptable. But then it should error out with helpful message when file is not there.

Thanks, I added a build optino and message.
Indeed, it is as you say, and I understand it is borderline acceptable.
I mainly wanted to check I'm not doing anything stupid, and though it may be useful to other users.

**Comment by dkwo** [comment]
One remark is that symlinks have permissions
```
ls -la /usr/bin/mathematica
lrwxrwxrwx 1 root root 40 Feb 15 11:34 /usr/bin/mathematica -> /opt/Mathematica/Executables/mathematica
```
while the actual files they point to have correct permissions.
I assumed this is fine, but please correct me.

**Pull request closed without merging**